.github/workflows/docker.yml

```diff
@@ -0,0 +1,44 @@
+name: Docker Image CI
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag'
+        required: true
+  release:
+    types: [created]
+    
+jobs:
+  build-and-push:
+    runs-on: github-hosted-heavy-runner
+    steps:
+
+    - 
+      name: Set up Docker Buildx
```

Comment:
I'm just curious why the dash and the `name` are on different lines? See line 37.

Reply:
No relevant reason. Both are parsed the same way. A matter of preference. I don't have strong opinion, we can change to `- name...` if you prefer

Reply:
It's up to you 😄